src/components/__tests__/SubThought.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import App from '../../App'
+import { getThoughtsRanked } from '../../util.js'
+
+// eslint-disable-next-line fp/no-let
+let wrapper = null
+
+beforeAll(async () => {
+  wrapper = await window.createTestApp(<App />)
+})
+
+afterAll(() => {
+  window.cleanupTestApp()
+  wrapper = null
+})
+
+it('create and edit subthought', async () => {
+  const thoughts = wrapper.find('div.transformContain div ul')
```

Comment:
`.transformContain` is not relevant for this test. We should try to write selectors that are semantically relevant for the current test.

Reply:
Same here, I tried.